users-react-app/src/pages/Layout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Header from '../components/Header/Header';
+
+function Layout(props){
```

Comment:
How about using an arrow function for this functional component? Like this:
```suggestion
export const Layout = (props) => {
```
And be done with the export in the same line too 😄

Reply:
I resolved it : 
`const Layout = (props) => {
  return (
    <>
      <Header />
      {props.children}
    </>
  );
};
export default Layout;`

How can you apply export in the same line? I've tried this way but it does not work:

`export const Layout = (props) => {
  return (
    <>
      <Header />
      {props.children}
    </>
  );
};
 `

Console throw this error:

![image](https://user-images.githubusercontent.com/21247095/120836447-f6d79d00-c53b-11eb-8ac2-26d85cc33c8b.png)

Reply:
You don`t need to add al the funcion in the same line 👀